### PR TITLE
Fix: Make chromium happy

### LINKED ISSUE #6892: [Bug] Chrome Extension: Unchecked runtime.lastError: The message port closed before a response was received.
# Bug Report

## Bug Description
Viewing any website/page causes an error in the console: 
```
Unchecked runtime.lastError: The message port closed before a response was received.
```

## Website Address
Any website.

## Steps To Reproduce
- open any website
- open the dev tools > console
- an error is written to the console  (see above)
- refresh the page; error is repeated

## Expected Behavior
No errors are thrown or written to the console.

## Actual Behavior
Error is written out to the console.

## Screenshots
n/a

## System Information
- OS: Windows 10 x64 
- Windows Version: 20H2
- Windows Build: 19042.1237
- Windows Feature Experience Pack 120.2212.3530.0
- Browser: Chrome Version 94.0.4606.54 (Official Build) (64-bit)
- Dark Reader Version: 4.9.37

## Additional Context
Toggled all my extensions off until I found the extension throwing the above error. Disabling Dark Reader and the error is no longer thrown. Enabling Dark Reader and the error is shown again.

## Possible Resolution?
- Stackoverflow: https://stackoverflow.com/a/56483156/9664
- Be sure to read the comments of the linked answer as well; accepted answer is crap.

### PR DESCRIPTION
**Pull request by Gusted**
- Fixes the `chrome.runtime.lastError` message.
- Ensure Chromium does like our current message system.
- Resolves #6892

### CODE REVIEW

**Comment by Gusted** [comment]
@bershanskiy or @alexanderby Could one of you have a look and merge it?

File: src/background/tab-manager.ts [hunk 1]
```diff
@@ -211,6 +211,8 @@ export default class TabManager {
                     break;
                 }
             }
+            sendResponse('Nothing-to-be-seen-here');
```
Comment (alexanderby) [inline src/background/tab-manager.ts:214]: Maybe worth sending something like {type: 'anything'}? Less chance to break anything, like object destructuring.

**Review by alexanderby — APPROVED** [review]

File: src/background/tab-manager.ts (hunk 1 above)
Comment (Gusted) [reply src/background/tab-manager.ts]: A lemmy face should do it :)

File: src/background/tab-manager.ts [hunk 2]
```diff
@@ -211,6 +211,8 @@ export default class TabManager {
                     break;
                 }
             }
+            sendResponse({type: '¯\\_(ツ)_/¯'});
```
Comment (alexanderby) [inline src/background/tab-manager.ts:214]: The most hilarious response 🤣

**Review by alexanderby — APPROVED** [review]

**Comment by bershanskiy** [comment]
I'm late, but technically we could write `return false;` since all calls to `sendResponse()` happen within the function. But that doesn't change anything AFAIK.

I'm curious why I didn't notice the error during development.

**Pull request merged**